<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="LAPTOP-9MQU4I3H" failures="1" tests="1" name="com.hrms.testscript.TC002" time="2.533" errors="0" timestamp="15 Mar 2024 18:42:49 GMT">
  <testcase classname="com.hrms.testscript.TC002" name="tc002" time="2.533">
    <failure message="msedgedriver located at C:\WINDOWS\system32\config\systemprofile\.cache\selenium\msedgedriver\win64\122.0.2365.92\msedgedriver.exe, but invalid
For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors/driver_location/
Build info: version: &#039;4.10.0&#039;, revision: &#039;c14d967899&#039;
System info: os.name: &#039;Windows 11&#039;, os.arch: &#039;amd64&#039;, os.version: &#039;10.0&#039;, java.version: &#039;20.0.1&#039;
Driver info: driver.version: EdgeDriver" type="org.openqa.selenium.remote.NoSuchDriverException">
      <![CDATA[org.openqa.selenium.remote.NoSuchDriverException: msedgedriver located at C:\WINDOWS\system32\config\systemprofile\.cache\selenium\msedgedriver\win64\122.0.2365.92\msedgedriver.exe, but invalid
For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors/driver_location/
Build info: version: '4.10.0', revision: 'c14d967899'
System info: os.name: 'Windows 11', os.arch: 'amd64', os.version: '10.0', java.version: '20.0.1'
Driver info: driver.version: EdgeDriver
	at org.openqa.selenium.remote.service.DriverFinder.getPath(DriverFinder.java:45)
	at org.openqa.selenium.edge.EdgeDriver.generateExecutor(EdgeDriver.java:70)
	at org.openqa.selenium.edge.EdgeDriver.<init>(EdgeDriver.java:59)
	at org.openqa.selenium.edge.EdgeDriver.<init>(EdgeDriver.java:55)
	at org.openqa.selenium.edge.EdgeDriver.<init>(EdgeDriver.java:47)
	at org.openqa.selenium.edge.EdgeDriver.<init>(EdgeDriver.java:43)
	at com.hrms.lib.General.openApplication(General.java:10)
	at com.hrms.testscript.TC002.tc002(TC002.java:15)
	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
	at java.base/java.lang.reflect.Method.invoke(Method.java:578)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:80)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1198)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1123)
	at org.testng.TestNG.run(TestNG.java:1031)
	at org.testng.TestNG.privateMain(TestNG.java:1338)
	at org.testng.TestNG.main(TestNG.java:1307)
]]>
    </failure>
  </testcase> <!-- tc002 -->
</testsuite> <!-- com.hrms.testscript.TC002 -->
